app/hallebarde/domain/exchange.py

```diff
@@ -11,6 +11,7 @@ class Exchange:
     download_token: str
     revoked_upload: bool = False
     creation_time: datetime = datetime.now(timezone.utc)
+    #email: str # mettre une regex par exple, se servir du package re : re.findall("[^\s()<>]+@[^\s()<>]+",content)
```

Comment:
I suggest that we do not validate the email format. Mainly for these arguments:https://www.loqate.com/resources/blog/3-reasons-why-you-should-stop-using-regex-email-validation/

And the fact that the email adress is provided to us by the OAuth server in a signed JWT, so therefore is sufficiently validated.